packages/platform/atoms/cal-provider/CalProvider.tsx

```diff
@@ -17,6 +20,8 @@ export type CalProviderProps = {
   autoUpdateTimezone?: boolean;
   onTimezoneChange?: () => void;
   version?: API_VERSIONS_ENUM;
+  labels?: Record<translationKeys, string>;
+  language?: "en" | "fr" | "pt-BR" | "de" | "es";
```

Comment:
`"en" | "fr" | "pt-BR" | "de" | "es"` can be defined in "CalProvider.tsx" file in "type AtomsLanguages = "en" | "fr" | "pt-BR" | "de" | "es"" that can be used here and also in `getTranslation`

Reply:
adding discriminative union for this